crt_portal/static/js/set_referrer_on_report.js

```diff
@@ -0,0 +1,19 @@
+(function(root) {
+  referrer = document.referrer;
+  host = document.location.host;
+  if (referrer.indexOf(host) > -1) {
+    referrer = '';
+  }
+
+  if (localStorage.getItem('referrer') == null && referrer != '') {
+    localStorage.setItem('referrer', referrer);
+  }
+
+  if (localStorage.getItem('referrer') != null && localStorage.getItem('referrer') != '') {
+    referrerField = document.getElementById("id_0-referrer");
+    if (referrerField) {
```

Comment:
Good idea to check referrerField. Should we check to see if localStorage.getItem('referrer') exists as well?  

Also, to avoid multiple calls to localStorage, we might want to set  localStorage.getItem('referrer') to a variable.

Reply:
> Good idea to check referrerField. Should we check to see if localStorage.getItem('referrer') exists as well?

I'm not sure what you're asking.  Do you think that the current check for `localStorage.getItem('referrer')` is not needed?  I include that conditional because I don't think we should update the referrer field if there's no referrer in local storage.

Reply:
Oh I misread the code.  You are right!  Ignore the first part of this comment.